### ISSUE TITLE
Support for tensor subtract?

### ISSUE THREAD

**Issue by ericwu09** [fchollet/keras#7570]
I've found a way to subtract two tensors in Keras:
`
input0 = Input(shape=(1, 512, 512))
layer_1 = Conv2D(64, (3, 3), activation='relu', padding='same')(input0)
layer_2 = Conv2D(64, (3, 3), activation='relu', padding='same')(layer_1)
diff = merge([layer_1, layer_2], mode=lambda x: x[0] - x[1], output_shape=lambda x: x[0])
`
But it looks like the merge function is being deprecated by the end of month. Are there ways with the new Merge Layers to perform subtraction?

**Comment by mahnerak** [comment]
You can simply use Lambda layer
```python
import numpy as np
from keras.models import Model
from keras.layers import Input, Lambda

x = Input(shape=(100,))
y = Input(shape=(100,))

# Note: shapes must match!
subtract_layer = Lambda(lambda inputs: inputs[0] - inputs[1],
                        output_shape=lambda shapes: shapes[0])
diff = subtract_layer([x, y])

model = Model(inputs=[x, y], outputs=diff)

output = model.predict([np.random.randn(32, 100), np.random.randn(32, 100)])
print(output)
```
